docs/conf.py

```diff
@@ -16,39 +16,39 @@
 import re
 import sys
 
-sys.path.insert(0, os.path.abspath('.'))
```

Comment:
these appear to be only formatting issues. since this is a big feature we're introducing, could you please open a new PR for this file?

Reply:
These are unwanted changes that I had missed on the diff. I will just revert them. Thanks for spotting them